### PR TITLE
Apply Spring Boot 3.4.0-RC1 #649

### PR DESCRIPTION
**Pull request by btkawaharahd**
This reverts commit bd0bde58dcd9432c0c4161faa2d966e5900dfbf0.

Please review #649.

### CODE REVIEW

**Comment by btkawaharahd** [comment]
https://terasolunaorg.atlassian.net/browse/TSF4J5-5356

File: testScript/parts/JavaConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java [hunk 1]
```diff
@@ -5,13 +5,16 @@
 
 import java.time.Duration;
 
+import org.junit.After;
```
Comment (btiwahorisu) [inline testScript/parts/JavaConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java:8]: - [ ] マルチプロジェクトと同様。
https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/pull/710#discussion_r1833675120

Comment (btkawaharahd) [reply testScript/parts/JavaConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java]: 403bd99
Removed unnecessary import statements.

File: testScript/parts/JavaConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java [hunk 2]
```diff
@@ -22,6 +22,7 @@
 @ContextConfiguration(classes = {SeleniumContextConfig.class})
 public class ErrorTest extends FunctionTestSupport {
 
+
```
Comment (btiwahorisu) [inline testScript/parts/JavaConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java:25]: - [ ] Remove unnecessary line breaks.

File: testScript/parts/XMLConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java [hunk 3]
```diff
@@ -5,21 +5,26 @@
 
 import java.time.Duration;
 
+import org.junit.After;
```
Comment (btiwahorisu) [inline testScript/parts/XMLConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java:8]: - [ ] I don't think any modifications are necessary to the ErrorTest on the XMLConfig side either.

File: testScript/parts/JavaConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java (hunk 2 above)
Comment (btkawaharahd) [reply testScript/parts/JavaConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java]: bd060d3
Corrected as noted.

File: testScript/parts/XMLConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java (hunk 3 above)
Comment (btkawaharahd) [reply testScript/parts/XMLConfig/projectName-selenium/src/test/java/xxxxxx/yyyyyy/zzzzzz/selenium/error/ErrorTest.java]: bd060d3
Corrected as noted.

**Review by btiwahorisu — APPROVED** [review]

**Pull request merged**